services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfeLinkBolt.java

```diff
@@ -588,8 +589,10 @@ private void handleSentDiscoPacket(DiscoPacketSendingConfirmation confirmation)
 
     @Override
     public void declareOutputFields(OutputFieldsDeclarer declarer) {
-        declarer.declareStream(SPEAKER_STREAM, new Fields("key", "message"));
-        declarer.declareStream(TOPO_ENG_STREAM, new Fields("key", "message"));
+        Fields fields = new Fields("key", "message");
```

Comment:
You should use `FieldNameBasedTupleToKafkaMapper.BOLT_KEY`, `FieldNameBasedTupleToKafkaMapper.BOLT_MESSAGE` instead as hardcoded `"key"`, `"message"`.